MANUAL.adoc

```diff
@@ -849,6 +849,27 @@ Most constants can be created once and reused as many times as needed.
 [id=const]
 ==== The `Const` interface
 
+The `Const` interface is used to represent expressions which are constant.
+This interface also contains factory methods to create new constants, including (but not limited to):
+
+* Primitive values (e.g. integers, booleans)
+* Primitive wrapper type instances (e.g. `Long`, `Double`)
+* `enum` constants (as <<jdk-const,`Constable`>> instances)
+* Strings
+* Classes (as `Class<?>` or as `ClassDesc`)
+* `null`
+* The value of a static final field
+* Lists of other object-typed constants (as an immutable `List`)
+* Maps of other object-typed constants (as an immutable `Map`)
+* Other <<jdk-const,JDK constant types>> such as method handles or dynamic constants
+
```

Comment:
Maybe also mention `Const.ofDefault()` below this list, as a way to obtain a default value of given type (zero for primitives, `null` for references)?